terraform/groups/ecs-service/variables.tf

```diff
@@ -0,0 +1,135 @@
+# ------------------------------------------------------------------------------
+# Environment
+# ------------------------------------------------------------------------------
+variable "environment" {
+  type        = string
+  description = "The environment name, defined in envrionment vars."
+}
+variable "aws_region" {
+  default     = "eu-west-2"
+  type        = string
+  description = "The AWS region for deployment."
+}
+variable "aws_profile" {
+  default     = "development-eu-west-2"
+  type        = string
+  description = "The AWS profile to use for deployment."
+}
+
+# ------------------------------------------------------------------------------
+# Docker Container
+# ------------------------------------------------------------------------------
+variable "docker_registry" {
+  type        = string
+  description = "The FQDN of the Docker registry."
+}
+
+# ------------------------------------------------------------------------------
+# Service performance and scaling configs
+# ------------------------------------------------------------------------------
+variable "desired_task_count" {
+  type        = number
+  description = "The desired ECS task count for this service"
+  default     = 1 # defaulted low for dev environments, override for production if required
+}
+variable "max_task_count" {
+  type        = number
+  description = "The maximum number of tasks for this service."
+  default     = 3
+}
+variable "required_cpus" {
+  type        = number
+  description = "The required cpu resource for this service. 1024 here is 1 vCPU"
+  default     = 1024 # defaulted minimum required for fargate services, override for production
```

Comment:
I would default the values for required CPU's and memory to those you have in CIDEV and overwrite in profiles for staging in LIVE. i would also amend the comment to reflect the information of matching the CPU and memory to what is currently there for the legacy service.

Reply:
thanks @Ngriffiths1  I have made the change.